### ISSUE TITLE
More specific CSS styles

### ISSUE THREAD

**Issue by bruceharrison1984** [nibdo/calend#17]
The included CSS has some very generic styles that end up overriding the styles within the rest of my application. For example:
```css
p {
  padding: 0;
  margin: 0; }

html {
  font-size: 10px;
  font-family: MONOSPACE, SansSerif, sans-serif, serif; }

button {
  padding: inherit;
  margin: 0;
  border: none;
  font: inherit;
  color: inherit;
  background-color: transparent;
  cursor: pointer;
  appearance: auto;
  letter-spacing: inherit;
  word-spacing: inherit;
  line-height: inherit;
  text-transform: none;
  text-indent: inherit;
  text-shadow: none;
  display: flex;
  text-align: inherit;
  align-items: inherit;
  box-sizing: inherit;
  background-color: transparent;
  border-width: inherit;
  border-style: inherit;
  border-color: inherit;
  border-image: inherit; }
```
Loading the Calend styles overrides the ones I am applying via Tailwinds, which dramatically alters my app in ways I don't really care for.

More specific named styles would stop this behavior (or using something like styled-components, which you hinted at already). AFAIK, the three above styles are the only element styles.

**Comment by nibdo** [comment]
Thanks, will fix it, sorry about that. I was already going to add prefix to class names so there won't be any conflicts.

**Comment by bruceharrison1984** [comment]
No problem man. Can you tell I'm excited about this 😆

**Comment by nibdo** [comment]
Fixed in version 0.0.3, thanks again :)
